pkg/config/defaults.go

```diff
@@ -47,6 +47,9 @@ func setDefaults(local bool) {
 	viper.SetDefault("auth.ui_config.login_failed_message", "The credentials don't match.")
 	viper.SetDefault("auth.ui_config.login_cookie_names", "internal_auth_session")
 
+	viper.SetDefault("auth.remote_authenticator.default_user_group", "Viewers")
```

Comment:
note that lakefs' default group is Developers, I think that we should be aligned with the non-remote authenticator configuration.